tests/functional/spec/feedback.spec.js

```diff
@@ -0,0 +1,36 @@
+import SchemaFeedbackPage from "../generated_pages/feedback/feedback.page";
+import SummaryPage from "../generated_pages/feedback/summary.page";
+
+import FeedbackPage from "../base_pages/feedback.page";
+import FeedbackSentPage from "../base_pages/feedback-sent.page";
+
+describe("Feedback", () => {
+  describe("Given I launch and complete the test feedback survey", () => {
+    before(() => {
+      browser.openQuestionnaire("test_feedback.json");
+      $(SchemaFeedbackPage.submit()).click();
+      $(SummaryPage.submit()).click();
+    });
+
+    it("When I try to submit without providing feedback, then I stay on the feedback page and get an error message", () => {
+      browser.url("/submitted/feedback/send");
```

Comment:
This should be using the current page's `url()` method which currently would be incorrect but would be fixed by addressing my comment about a new base page for feedback.